platform/windows/key_mapping_win.cpp

```diff
@@ -236,11 +236,52 @@ VK_PA1 (0xFD)
 VK_OEM_CLEAR (0xFE)
 */
 
-unsigned int KeyMappingWindows::get_keysym(unsigned int p_code) {
+unsigned int KeyMappingWindows::get_keysym(unsigned int wParam, unsigned int lParam) {
+
+	unsigned int virtual_key = wParam;
+	static bool lshift_down = false;
```

Comment:
I have no issues with merging this PR, maybe not right away, so that other contributors might add linux/osx integrations into the mix.